src/Tribe/Aggregator/Tabs/New.php

```diff
@@ -559,6 +559,45 @@ public function maybe_display_aggregator_upsell() {
 		return ob_get_clean();
 	}
 
+	/**
+	 * Renders the "Eventbrite Tickets" upsell
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	public function maybe_display_eventbrite_upsell() {
+
+		if ( ! tribe( 'events-aggregator.main' )->is_service_active() ) {
+			return;
+		}
+
+		if ( class_exists( 'Tribe__Events__Tickets__Eventbrite__Main' ) ) {
+			return;
+		}
+
+		ob_start();
+		?>
+		<div class="tribe-dependent tribe-banner tribe-banner-eventbrite-tickets" data-depends="#tribe-ea-field-origin" data-condition="eventbrite">
```

Comment:
Why not shove the HTML into a view?

Reply:
We can do that. I tried to use the same logic as the other upsell banner to keep uniformity.